pages/Documents/Reporting/OutboundConfigurationAPI/overview.md

```diff
@@ -0,0 +1,400 @@
+| accountId | LivePerson site ID | Yes      | 12345678                                                                         |
+| app       | App name           | Yes      | "prmsg"                                                                          |
+| v         | Version            | Yes      | 1                                                                                |
+
+**Query Parameters**
+
+| Name   | Description                                   | Required | Value/Example |
+| :----- | :-------------------------------------------- | :------- | :------------ |
+| limit  | Max count of outbound configurations to fetch | No       | 30            |
+| offset | Starting point to fetch from                  | No       | 0             |
+
+**Request Headers**
+
+| Header        | Description                                                                                                                                                                                                                                         | Value/Example    |
+| :------------ | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | :--------------- |
+| Content-Type  | Used to indicate the media type of the resource                                                                                                                                                                                                     | application/json |
+| Authorization | [OAuth 2.0](https://developers.liveperson.com/connector-api-send-api-authorization-and-authentication.html#get-appjwt) or [OAuth 1.0](https://developers.liveperson.com/retrieve-api-keys-create-a-new-api-key.html) (Section 8) or LE Bearer token |                  |
+
+**Response Example**
+
+200 Success
+
+* ​sms-twilio:<br><br>
+  ​sms-twilio response example:
+```json
+{
+   "type": "sms-twilio",
+   "language": "en",
+   "outboundnumber": "+14156586515",
+   "messageComponents": [
+        {
+        "type": "body",
+        "content": "Hi ${{1}}",
+        "variables": ["1"],
+        "optOut": "To stop receiving messages, reply STOP. For help, reply HELP."
+        }
+   ]
+}
+```
+
+* WhatsApp:<br><br>
+  WhatsApp response example:
+```json
+​{
+   "type": "wa",
+   "language": "en",
+   "namespace": "b8520e15_f77b_5f76_fb50_9f756d03676b",
+   "templatename": "newtemplate_button_cta_feb14",
+   "outboundnumber": "12535277322",
+   "messageComponents": [
+       {
+           "type": "body",
+           "content": "Your ticket for ${{1}}\n\nTime - ${{2}}\nVenue - ${{3}}\nSeats - ${{4}} - ${{5}}",
+           "variables": [
+               "1",
+               "2",
+               "3",
+               "4",
+               "5"
+           ]
+       }
+  ]
+}
+```
+  WhatsApp rich content response example
+```json
```

Comment:
All code: Please indent by 2 spaces, and do so consistently.

If you look at the `"type": "body"` line, for example, this is being indented to a degree that it’s probably unusable on a mobile screen.

Reply:
@j9t Do you mean, we need to indent all the lines of JSON by just 2 spaces? Because I think that could make JSON unreadable for all users.

Reply:
Not sure what you have in mind, but I mean standard indentation, just every time you indent, indent by 2 spaces. For example, line 133 (the type/body one), instead of indenting by 34 spaces, indent by 6 spaces.

Reply:
Done